extensions/amp-image-viewer/0.1/amp-image-viewer.js

```diff
@@ -275,19 +276,55 @@ export class AmpImageViewer extends AMP.BaseElement {
     }
   }
 
+  /**
+   * @return {number}
+   * @private
+   */
+  getSourceWidth_() {
+    if (this.sourceAmpImage_.hasAttribute('width')) {
+      return parseInt(this.sourceAmpImage_.getAttribute('width'), 10);
+    } else {
+      const img = elementByTag(dev().assertElement(this.sourceAmpImage_),
+          'img');
+      if (img) {
```

Comment:
nit: In the rest of the codebase we simplify `if` statements like this to ternaries.